spec/core/SpyStrategySpec.js

```diff
@@ -92,16 +92,40 @@ describe("SpyStrategy", function() {
     expect(returnValue).toEqual(67);
   });
 
+  it("allows a fake async function to be called instead", async function(done) {
+    try {
+      var originalFn = jasmine.createSpy("original"),
+          fakeFn = jasmine.createSpy("fake").and.callFake(async function () { return 67; }),
```

Comment:
Using `async` here is fine if you guard the spec with `jasmine.getEnv().requireAsyncAwait()` so that this part won't execute on platforms that don't support it.